### DISCUSSION TITLE
Working fine and w/o issues on openSUSE Tumbleweed

### DISCUSSION THREAD

**Discussion by RafaelLinux** [ivan-hc/Steam-appimage#45] [General]
Great work. I'm playing now ZeroK but will try another games. But simply get it to launch w/o any issues, is really to congratulate you on the work.

Thanks

**Comment by ivan-hc** [comment]
thanks, this was not possible without the help of @Samueru-sama

**Comment by Samueru-sama** [comment]
Thanks Rafael.

**Comment by RafaelLinux** [comment]
To you, of course, I am a simple user.
To try to keep the information up to date, I will comment on any issues here as long as possible.
At the moment, using ZeroK for several hours, I noticed that the game “froze” up to 3 or 4 times, for between 2 and 7 seconds. But as it only happened to me playing online, I can't relate it directly to the AppImage.

**Reply by ivan-hc** [reply]
it is better you open the issues in "issues" https://github.com/ivan-hc/Steam-appimage/issues because discussions are not more visible and people who have issues can rely on the dedicated section

**Reply by Samueru-sama** [reply]
The freezing might be due to shaders being compiled, since the appimage ships its own video drivers. 

it may also be due to higher ram usage, which the appimage has, however this only really is a problem if you have less than 8 GIB of RAM.

**Comment by RafaelLinux** [comment]
I have 32GiB. Anyway, I do not consider neccessary to open issue while not checked this problem enough.

**Comment by RafaelLinux** [comment]
Yesterday I had zero "freezes" with ZeroK. Time to check Tomb Raider.
